### PR TITLE
[ECS][Filebeat] Gsuite/Google Workspace ECS 1.8

### PR DESCRIPTION
**Pull request by marc-gr**
## What does this PR do?

Add multiuser and new categories to gsuite and google_workspace modules

## Why is it important?

Keep the modules up to date with ECS 1.8 changes

## Checklist

~- [ ] My code follows the style guidelines of this project~
~- [ ] I have commented my code, particularly in hard-to-understand areas~
~- [ ] I have made corresponding changes to the documentation~
- [x] I have made corresponding change to the default configuration files
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.

### CODE REVIEW

**Review by andrewkroh — COMMENTED** [review]
Does this include changes from other open PRs that it's dependent upon?

**Comment by andrewkroh** [comment]
The changes in 1f3e733c8ad8975e7f4443809fdcbaa683621683 LGTM.

**Comment by marc-gr** [comment]
> Does this include changes from other open PRs that it's dependent upon?

had to rebase the feature branch

File: CHANGELOG.next.asciidoc [hunk 1]
```diff
@@ -836,6 +836,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Added `application/x-www-form-urlencoded` as encode option for httpjson input {pull}23521[23521]
 - Added RFC6587 framing option for tcp and unix inputs {issue}23663[23663] {pull}23724[23724]
 - Upgrade Cisco ASA/FTD/Umbrella to ECS 1.8.0. {pull}23819[23819]
+- Add new ECS user and categories features to google_workspace/gsuite {pull}23118[23118]
```
Comment (adriansr) [inline CHANGELOG.next.asciidoc:839]:
```suggestion
- Add new ECS user and categories features to google_workspace/gsuite {issue}23118[23118] {pull}23709[23709]
```

**Review by adriansr — APPROVED** [review]
LGTM.


We can ignore the PR numbers in CHANGELOG for now and adjust them before merging the ecs 1.8 branch.

**Pull request merged**